dashboard/db/migrate/20160901180359_drop_activity_hint.rb

```diff
@@ -0,0 +1,7 @@
+class DropActivityHint < ActiveRecord::Migration[5.0]
```

Comment:
No, I do not believe so. To start, old migrations should never need to run. And, even should we run them, it is only deprecated (not yet unsupported).